src/pages/signIn/SignInPage.tsx

```diff
@@ -1,13 +1,37 @@
 import React from 'react';
 
+import { useCookies } from 'react-cookie';
+import Select from 'react-select';
+
 import SignInForm from 'components/forms/signInForm';
 
 export default function SignInPage() {
+  const [langCookies, setLangCookie] = useCookies(['language']);
+
+  const options = [
+    { value: 'en', label: 'English' },
```

Comment:
maybe we can take advantage of having const from 'utils/consts' if you fix consts related to this values.

Reply:
This is use only in one place